pylint/checkers/imports.py

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
I think the correct pragma is `# -*- coding: utf-8 -*-

Reply:
There are several: https://www.python.org/dev/peps/pep-0263/. I use `# -*- coding: utf-8 -*- ` because Sphinx also recognizes it.

Reply:
Thanks @Pawamoy ! I wasn't aware of that